vendor/bat-native-confirmations/src/bat/confirmations/internal/logging_util.cc

```diff
@@ -13,9 +13,20 @@
 
 namespace confirmations {
 
-namespace {
+bool ShouldLogHeader(const std::string& header) {
+  const std::vector<std::string> excluded_headers = {
```

Comment:
Can we please change the vector to use `std::set` and `lower_bound`, i.e.
```
const std::set<std::string> headers{
  "authorization",
  "cookie"
};

std::string prefix(header);

const auto iter = headers.lower_bound(prefix);
if (iter == headers.end() {
  return false;
}

return true;
```

Reply:
sadly code doesn't work so we can change it later on. It was discussed on the call